apps/api/src/queries/term/term-queries-fetch-by-id.e2e.spec.ts

```diff
@@ -189,7 +189,7 @@ describe(`when querying for a term: fetch by Id`, () => {
 
                 /**
                  * TODO We need a contract test that will warn us if we potentially
-                 * break the client via breaking chagnes to the API.
+                 * break the client via breaking changes to the API.
```

Comment:
Do we snapshot one successful query response? That should do it assuming our test data is comprehensive (includes all optional props).

Reply:
Let me look into this once you've finished the other refactors.

Reply:
We **do** check one snapshot, for the project admin happy path case.